conan/tools/cmake/toolchain/blocks.py

```diff
@@ -502,6 +505,38 @@ class FindFiles(Block):
         {% endif %}
     """)
 
+    @staticmethod
+    def _multiconfig_generator(prop):
+        return ' '.join(f'"$<$<CONFIG:{c}>:{i}>"' for c, v in prop.items() for i in v)
```

Comment:
I think the way it is now it's correct, however since we can check for `is_multi_configuration(self._toolchain.generator)` - we probably dont need a generator expression if we know the generator is single config